realm/realm-library/src/main/java/io/realm/Mixed.java

```diff
@@ -19,537 +19,379 @@
 import org.bson.types.Decimal128;
 import org.bson.types.ObjectId;
 
-import java.util.Collections;
 import java.util.Date;
 import java.util.UUID;
 
+import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
-import io.realm.internal.ManageableObject;
-import io.realm.internal.OsSharedRealm;
-import io.realm.internal.RealmProxyMediator;
-import io.realm.internal.Row;
-import io.realm.internal.Table;
-
 
 /**
- * {@link Mixed} is used to represent a polymorphic Realm value.
- * It has two modes: a managed and unmanaged mode. In managed mode contents are persisted inside a Realm, in
- * unmanaged mode contents are persisted in the object instance.
+ * {@link io.realm.Mixed} is used to represent a polymorphic Realm value.
+ * <p>
```

Comment:
Maybe also add a section about this being a value type and what implications that have?